cli/impl/src/test/java/io/helidon/build/cli/impl/TestUtils.java

```diff
@@ -188,6 +224,31 @@ static String helidonTestVersion() {
         return version;
     }
 
+    /**
+     * Get the Helidon shell script Path.
+     *
+     * @return script path
+     * @throws IllegalStateException if the {@code helidon.shell.script} is system property not found
+     */
+    static Path helidonShellScript() {
+        String script = System.getProperty("helidon.script");
+        if (script == null) {
+            throw new IllegalStateException("Unable to resolve helidon.script system property");
+        }
+        script = OSType.currentOS().equals(OSType.Windows)
+                ? script.concat(".bat")
+                : script.concat(".sh");
+        return Path.of(script);
+    }
+
+    static Path helidonNativeImage() {
+        String script = System.getProperty("helidon.script");
```

Comment:
If same property name is used for both script and native image then the term `script` would be better replaced with `executable`